packages/web-components/src/components/va-accordion/va-accordion-item.tsx

```diff
@@ -52,6 +52,11 @@ export class VaAccordionItem {
    */
   @State() slotTag: string = null;
 
+  /**
+   * Toggle button reference
+   */
+  private expandButton: HTMLElement = null;
```

Comment:
This can be changed to HTMLButtonElement to better represent the type.

Reply:
Done, thanks1